app/response_processor.py

```diff
@@ -128,14 +114,33 @@ def make_receipt(self, decrypted_json):
 
         return receipt_json
 
+    def send_to_downstream(self, decrypted_json):
+        if 'invalid' in decrypted_json:
```

Comment:
Is it maybe worth looking at another way of doing this? Say we get x, y, z different types of survey, will we need to add another `elif x, elif y, elif z` for each one?

Reply:
I agree with Darrel the cyclomatic compexity seem quite high here.